.changeset/dull-walls-lie.md

```diff
@@ -0,0 +1,5 @@
+---
+"@alephium/mobile-wallet": patch
```

Comment:
Switching networks leads to this issue:

<img width="350" alt="image" src="https://github.com/user-attachments/assets/fb5a47b4-1a53-46a2-8d1a-8ddff1132247" />
<img width="350" alt="image" src="https://github.com/user-attachments/assets/c2972c71-22ef-4f6b-92e6-5e15ff9a67dc" />

Reply:
I'm wokring on it